### ISSUE TITLE
[BUDI-6551] MySQL - Options (and Long Form Text) field resets itself to text type (and cannot be changed) after fetching tables into new app 

### ISSUE THREAD

**Issue by Darojax** [Budibase/budibase#9602]
**Hosting**
- Self
  - Method: docker compose
  - Budibase Version: 2.2.23
  - MariaDB 10.10.2

**Describe the bug**
When fetching tables into a new app 'Options' (and 'Long Field' Text) Columns are displayed as 'Text' type columns. The Options that had been entered previously and still work in the original app from where the column was created still works, and running a query shows that it's still an Enum column. But in the new app where the tables were fetched it shows as 'text' type. It's not possible to change the type shown in Budibase, nor is it possible to change the name of this column. I've tried deleting the column and recreating it as Options type from scratch, but still when fetching this column into a new app, the problem returns. Is this a bug or something I am doing wrong?

**To Reproduce**
1. Create an Options type column and enter some options.
2. Create a new App and fetch the tables with the Option column.
3. In the new App the Options column is now a text column, which cannot be name changed or type changed, and all the options are gone.
4. Deleting and recreating the column does not help. Both apps are running on the latest Budibase version 2.2.23

**Expected behavior**
Fetching the table should retain the Options type column.

**App Export**
Apps attached. the Recruit Status app is where the Options column was created and works fine, it's the personnel.status column. The ORBAT app is where those tables have been fetched and the personnel.status Options column no longer works as expected, it's text type and cannot be changed.

**Desktop (please complete the following information):**
 - OS: Windows 11 Pro
 - Browser Chrome
 - Version 109.0.5414.120

[Recruit Status-export-1675903251875.tar.gz](https://github.com/Budibase/budibase/files/10691354/Recruit.Status-export-1675903251875.tar.gz)
[ORBAT-export-1675903276200.tar.gz](https://github.com/Budibase/budibase/files/10691355/ORBAT-export-1675903276200.tar.gz)

<sub>[BUDI-6551](https://linear.app/budibase/issue/BUDI-6551/mysql-options-and-long-form-text-field-resets-itself-to-text-type-and)</sub>

**Comment by mike12345567** [comment]
Hi @Darojax - in the latest version I've been unable to re-create this issue - can try updating to `2.3.12`.

**Comment by Darojax** [comment]
Hello @mike12345567 , thanks for getting back on this.

It does appear as if now it is indeed possible to alter the type of the column, I was able to alter the Text type to Options type. However, re-fetching the table still does not produce an 'Options' type table, I have to make the change manually. I have verified that the column type is 'Enum' with several options using the "DESCRIBE table_name;" query, but when refetching the table it still remains 'text'.

But now the problem is no longer critical, thanks again. : )
